dbt_project.yml

```diff
@@ -23,12 +23,23 @@ clean-targets:         # directories to be removed by `dbt clean`
 
 models:
   dbt_project_evaluator:
-    dag:
-      +materialized: table
-    graph:
-      intermediate:
-        int_direct_relationships:
-          +materialized: "{{ 'table' if target.type == 'bigquery' else 'view' }}"
+    marts:
+      dag:
+        +materialized: table
+    staging:
+      graph:
+        intermediate:
+          int_direct_relationships:
+            # required for BigQuery and Redshift for performance/memory reasons
+            +materialized: "{{ 'table' if target.type in ['bigquery', 'redshift'] else 'view' }}"
+        stg_naming_convention_folders:
```

Comment:
Should we move `stg_naming_convention_folders` and `stg_naming_convention_prefixes` into a separate folder since they're not related to the graph object? Maybe something like `vars`?

Reply:
@graciegoheen , I looked at it and I think that there is a bit more we have to do:
- create a `structure` folder under `staging` for those models (same name as the folder under `marts`)
- change `stg_all_graph_resources` to `int_all_graph_resources` and move it under `marts/.../int`
- move models from `staging/graph/intermediate` to `marts/dag/intermediate` (or similar)

I'd rather get the current PR over the line (there are quite a few changes in here) and deal with the structure changes in another issue. Good with you?

Reply:
Agree with this comment, but also agree it's not blocking! I also think since these files are mostly different in terms of configs we might want to use the `{{ config() }}` in each model?

Reply:
Sounds good! Thanks for making a list!